### PR TITLE
Es6 modules

### PR DESCRIPTION
**Pull request by rkirov**

### CODE REVIEW

**Comment by rkirov** [comment]
cc @rictic

File: src/test/java/com/google/javascript/clutz/partial/es6_module.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import * as rel from './rel/module/name';
+import * as rel2 from '/abs/module/name';
+import * as abs from 'abs_strip_for_testing/module/name';
+
+export let x = rel.x + abs.x + rel2.x;
+
+//!! The emit of this file is intentionally empty.
+//!! So far clutz doesn't support es6 modules properly, but the test
+//!! makes sure that it doesn't reject them either.
```
Comment (evmar) [inline src/test/java/com/google/javascript/clutz/partial/es6_module.js]: Trailing newline

File: src/test/java/com/google/javascript/clutz/partial/es6_module.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import * as rel from './rel/module/name';
+import * as rel2 from '/abs/module/name';
+import * as abs from 'abs_strip_for_testing/module/name';
```
Comment (evmar) [inline src/test/java/com/google/javascript/clutz/partial/es6_module.js]: Maybe doc that 'abs_strip_for_testing' is special-cased in the test harness?

Comment (rkirov) [reply src/test/java/com/google/javascript/clutz/partial/es6_module.js]: done

File: src/test/java/com/google/javascript/clutz/partial/es6_module.js (hunk 1 above)
Comment (rkirov) [reply src/test/java/com/google/javascript/clutz/partial/es6_module.js]: done

**Pull request merged**